backend/indexer/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import { ponder } from "@/generated";
+import { BigNumber } from "ethers";
+
+ponder.on("EtherspotPaymaster:SponsorSuccessful", async ({ event, context }) => {
+  const { db, network } = context;
+  const { args, block, transaction, log } = event;
+  const time = Number(BigNumber.from(block.timestamp).toString());
+  const timestamp = new Date(time * 1000);
+
+  console.log('saved event: ', event);
```

Comment:
Please use a javascript logger package instead of console.log

Reply:
that was for debugging should be removed. Will remove that